### ISSUE TITLE
Warnmeldungen im iob log.

### ISSUE THREAD

**Issue by bahnuhr** [basti4557/ioBroker.komoot#4]
im log kommt:

<html>
<body>

komoot.0 | 2024-02-09 13:08:40.089 | warn | Status 302: www.komoot.de/user/28xxx/tours?type=recorded
komoot.0 | 2024-02-09 13:08:40.288 | warn | Status 302: www.komoot.de/user/28xxx/following
komoot.0 | 2024-02-09 13:08:40.220 | warn | Status 302: www.komoot.de/user/28xxx/followers
komoot.0 | 2024-02-09 13:08:40.153 | warn | Status 302: www.komoot.de/user/28xxx

</body>
</html>

bitte Warn bereinigen.

**Comment by basti4557** [comment]
Ich sehe leider keine Lösung momentan für dieses Problem. Komoot hat ReCaptcha in sein Login eingefügt. Da die API nicht öffentlich bereitgestellt wird, wurde sich immer mittels der normalen Anmeldemethoden bei Komoot authorisiert. Eventuell könnte hier auf eine Art "virtuellen Browser" zurückgegriffen werden, was jedoch fehlschlägt, sofern der ReCaptcha einen Code verlangt.

Falls du eine Idee hast, lasse ich mich aber gerne eines Besseren belehren :) Wäre wirklich schade um den Adapter

**Comment by Gaspode69** [comment]
@basti4557 
Wenn ich das richtig verstehe, hat Ingo das beim Amazon Adapter mit einem lokalen Proxy gelöst, der sich die erzeugten Cookies merkt und das Login dann darüber gemacht. die Details kenne ich nicht und evtl. ist es ja auch ne ganz andere Problematik, aber evtl. könntest du dir mal ansehen, ob das auch für diesen Adapter ein gangbarer Weg wäre.

**Comment by bahnuhr** [comment]
@basti4557 
Hast du Ingo gefragt ?
Was hat er gesagt ?

Wäre doch gut wenn wir hier weiter kommen.
Ansonsten haben wir einen guten Adapter weniger.

**Comment by Gaspode69** [comment]
Nein, hab ihn nicht gefragt. Evtl. kann sich @basti4557 das mit dem Login ja dort "abgucken", falls es passen sollte. Ingo hat genug um die Ohren, da würde ich persönlich nur nachfragen, wenn es gar nicht anders geht.

**Comment by basti4557** [comment]
Leider sind die Cookies von Komoot unter einem Tag gültig, d.h. es müsste sich wie z.b. im Alexa Adapter jeden Tag mit der Proxy Anmeldemaske angemeldet werden, was nicht wirklich praktikabel ist.

Ich sehe keine Lösung für das Problem. Ich lass das hier mal noch etwas offen, falls vielleicht jemand auf etwas schlaues kommt.
Aber ansonsten werde ich das Projekt wohl archivieren müssen.

**Comment by bahnuhr** [comment]
Schade, dann ist der Adapter wohl wirklich nicht mehr zu gebrauchen.
Hast du denn mal bei @apollon gefragt. Vielleicht hat er eine Lösung.

**Comment by Gaspode69** [comment]
> Leider sind die Cookies von Komoot unter einem Tag gültig, d.h. es müsste sich wie z.b. im Alexa Adapter jeden Tag mit der Proxy Anmeldemaske angemeldet werden, was nicht wirklich praktikabel ist.

Bist du sicher? Ich hab das nicht weiter untersucht, aber in meinem Browser muss ich mich eigentlich nie bei Komoot neu anmelden, auch nicht nach Wochen.

**Comment by basti4557** [comment]
@Gaspode69 Ich werde mal bei mir lokal die Tage die Funktion des erneuten Logins jeden Tag deaktivieren und die Cookies von Komoot dann manuell abgreifen und im Adapter eintragen und schauen was passiert bzw. wie lange es funktioniert. Die meisten Cookies die ich hier sehe (Rechtsklick -> Untersuchen -> Application -> Cookies) sind jedoch nur Stunden gültig. Dazu sind noch 2 Cookies von der Session vorhanden, wo ich jetzt keine Lifetime sehe, die wird wohl auf dem Server konfiguriert sein. Bin aber leider noch in den USA somit wird das vermutlich noch bis nächste Woche dauern.

**Comment by bahnuhr** [comment]
@basti4557 
Hallo,
gibts hier was Neues ?
mfg
